### ISSUE TITLE
My running log is like this, why is there no win and TXS?

### ISSUE THREAD

**Issue by caitoudo** [gensyn-ai/rl-swarm#312]
'rewards/final_correctness_reward_func/std': 0.0, 'rewards/strict_format_reward_func/mean': 0.0, 'rewards/strict_format_reward_func/std': 0.0, 'rewards/soft_format_reward_func/mean': 0.0, 'rewards/soft_format_reward_func/std': 0.0, 'rewards/xmlcount_reward_func/mean': 0.0, 'rewards/xmlcount_reward_func/std': 0.0, 'rewards/cumulative_reward_2/mean': 0.0, 'rewards/cumulative_reward_2/std': 0.0, 'reward': 0.06794615807011724, 'reward_std': 0.010441619961056858, 'kl': 0.5806339549541008, 'clip_ratio/low_mean': 0.0, 'clip_ratio/low_min': 0.0, 'clip_ratio/high_mean': 0.0, 'clip_ratio/high_max': 0.0, 'clip_ratio/region_mean': 0.0, 'epoch': 0.4}
{'loss': 0.0137, 'grad_norm': 17.561965942382812, 'learning_rate': 3.4096741493194196e-08, 'num_tokens': 13303.0, 'completions/mean_length': 84.225, 'completions/min_length': 65.4, 'completions/max_length': 93.9, 'completions/clipped_ratio': 0.775, 'completions/mean_terminated_length': 21.5, 'completions/min_terminated_length': 17.4, 'completions/max_terminated_length': 26.2, 'rewards/consensus_reward_func/mean': 0.0, 'rewards/consensus_reward_func/std': 0.0, 'rewards/concensus_correctness_reward_func/mean': 0.0, 'rewards/concensus_correctness_reward_func/std': 0.0, 'rewards/question_recreation_reward_func/mean': 0.08099816534668207, 'rewards/question_recreation_reward_func/std': 0.03161046179011464, 'rewards/final_correctness_reward_func/mean': 0.0, 'rewards/final_correctness_reward_func/std': 0.0, 'rewards/strict_format_reward_func/mean': 0.0, 'rewards/strict_format_reward_func/std': 0.0, 'rewards/soft_format_reward_func/mean': 0.0, 'rewards/soft_format_reward_func/std': 0.0, 'rewards/xmlcount_reward_func/mean': 0.0011500000022351743, 'rewards/xmlcount_reward_func/std': 0.0023000000044703485, 'rewards/cumulative_reward_2/mean': 0.0, 'rewards/cumulative_reward_2/std': 0.0, 'reward': 0.08214816264808178, 'reward_std': 0.026475772843696177, 'kl': 0.3413976773619652, 'clip_ratio/low_mean': 0.0, 'clip_ratio/low_min': 0.0, 'clip_ratio/high_mean': 0.0, 'clip_ratio/high_max': 0.0, 'clip_ratio/region_mean': 0.0, 'epoch': 0.8}
{'train_runtime': 2034.7742, 'train_samples_per_second': 0.039, 'train_steps_per_second': 0.01, 'train_loss': 0.018440486490726472, 'epoch': 0.8}
100%|██████████████████████████████████████████████████████████| 20/20 [33:54<00:00, 101.74s/it]
***** train metrics *****
  total_flos               =        0GF
  train_loss               =     0.0184
  train_runtime            = 0:33:54.77
  train_samples            =         50
  train_samples_per_second =      0.039
  train_steps_per_second   =       0.01
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:meek hulking wolf:Saving model
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:meek hulking wolf:Model saved to runs/gsm8k/multinode/Qwen2.5-0.5B-Instruct-Gensyn-Swarm-meek_hulking_wolf
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:meek hulking wolf:Tokenizer saved to runs/gsm8k/multinode/Qwen2.5-0.5B-Instruct-Gensyn-Swarm-meek_hulking_wolf
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:meek hulking wolf:📉 Finished training round: 439 stage: 2
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:meek hulking wolf:🏆 Submitting winners for round 439: ['QmScsipAg3tFiLGwnYCH93KnhnuiEgBerdJZFyLpWA2tLX']
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:meek hulking wolf:🐝 Joining round: 440 starting at stage: 0
INFO:hivemind_exp.trainer.hivemind_grpo_trainer:meek hulking wolf:📈 Training round: 440 stage: 0

![Image](https://github.com/user-attachments/assets/7608f910-f652-4a18-a2c8-102e8febab6e)

**Comment by Aweeefg** [comment]
similar problem for me
